packages/@cdktf/hcl2cdk/test/__snapshots__/iteration.test.ts.snap

```diff
@@ -27,40 +27,54 @@ class MyConvertedCode extends cdktf.TerraformStack {
       });
     /*This allows the Terraform resource name to match the original name. You can remove the call if you don't need them to match.*/
     dataAwsRoute53ZoneExample.overrideLogicalId(\\"example\\");
+    /*In most cases loops should be handled in the programming language context and 
+    not inside of the Terraform context. If you are looping over something external, e.g. a variable or a file input
+    you should consider using a for loop. If you are looping over something only known to Terraform, e.g. a result of a data source
+    you need to keep this like it is.*/
+    const awsRoute53RecordExampleForEachIterator =
+      cdktf.TerraformIterator.fromList(
```

Comment:
In general we should be using e.g. `fromMap` here as well, but we have little chance to understand if the input value is closer to a list or a map. I was thinking about adding this to the comment to let the user change the code slightly if needed